deployment/run.bat

```diff
@@ -1,8 +1,8 @@
+@echo off
 REM --------------------------------------------------------------------------------------------
 REM Copyright (c) Microsoft Corporation. All rights reserved.
 REM Licensed under the MIT license.
 REM --------------------------------------------------------------------------------------------
 
-@echo off
```

Comment:
Any specific reason for removing this?

Reply:
It's not removed. Just moved to the top of the file. Without this all the `REM` comments on lines 2-4 were ending up getting displayed on the console.

Reply:
My bad!